webapp/view/PageOverview.view.xml

```diff
@@ -0,0 +1,203 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+	xmlns:f="sap.f"
+	xmlns:core="sap.ui.core"
+	xmlns:fb="sap.ui.comp.filterbar"
+	xmlns:layout="sap.ui.layout"
+	xmlns:svm="sap.ui.comp.smartvariants"
```

Comment:
I don't see you use this dependency anywhere. Do we really need it?

Reply:
No, was removed